go/pkg/pwapi/api_challenge-subscription-validate.go

```diff
@@ -63,6 +63,26 @@ func (svc *service) ChallengeSubscriptionValidate(ctx context.Context, in *Chall
 		return nil, errcode.ErrGetChallengeValidation.Wrap(err)
 	}
 
+	var seasonChallenge pwdb.SeasonChallenge
```

Comment:
fyi, seasonchallenge was already loaded earlier in the function (https://github.com/pathwar/pathwar/pull/453/files#diff-3f2512229937337814ba64f5d550c0f2R26)

you can call `challengeSubscription.SeasonChallenge`